odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/AlertRepositoryImpl.java

```diff
@@ -119,6 +125,68 @@
             .collect(Collectors.toList());
     }
 
+    @Override
+    public Page<AlertDto> listDependentObjectsAlerts(final int page, final int size, final long ownerId) {
```

Comment:
It seems you have recursive CTE inside another CTE. Let's have a chained CTEs and not nested